scripts/systemd/nutshutdown.in

```diff
@@ -1,2 +1,14 @@
 #!/bin/sh
-@SBINDIR@/upsmon -K >/dev/null 2>&1 && @SBINDIR@/upsdrvctl shutdown
+
+if @SBINDIR@/upsmon -K >/dev/null 2>&1; then
+  wait_delay=`/bin/sed -ne 's#^ *POWEROFF_WAIT= *\(.*\)$#\1#p' /etc/nut/nut.conf`
+
+  @SBINDIR@/upsdrvctl shutdown
+
+  if [ -n "$wait_delay" ] ; then
```

Comment:
it's also possible to pass strings here, like "15m" so I don't think it's possible